rustls/tests/server_cert_verifier.rs

```diff
@@ -7,9 +7,8 @@ use crate::common::{
     do_handshake, do_handshake_until_both_error, make_client_config_with_versions,
     make_pair_for_arc_configs, make_server_config, ErrorFromPeer, ALL_KEY_TYPES,
 };
-use rustls::client::{
-    HandshakeSignatureValid, ServerCertVerified, ServerCertVerifier, WebPkiVerifier,
-};
+use rustls::client::WebPkiVerifier;
```

Comment:
This seems unrelated? I'd rather keep all imports from the same module on the same line.